src/index.coffee

```diff
@@ -6,45 +6,39 @@ BrowserTransport = require './transports/browser-transport'
 CliTransport = require './transports/cli-transport'
 Logger = require './logger'
 
-
-getGlobalScope = ->
-  if typeof global != 'undefined'
-    return global
-
-  return window
-
-
-isBrowser = (globalScope) ->
-  return (
-    globalScope.window?.document?.createElement and
-    globalScope.window?.name != 'nodejs'
-  )
+{ getGlobalScope, isBrowser } = require './utils'
+{ setupTransport } = require './setup-transport'
 
 
 module.exports =
   LogLevels: LogLevels
   
   Logger: Logger
 
+  BrowserTransport: BrowserTransport
+  CliTransport: CliTransport
+
   create: (options = {}) ->
     globalScope = getGlobalScope()
     browser = isBrowser(globalScope)
 
     if browser
-      transport = new BrowserTransport(globalScope.console)
-      transport.setLogLevelPrefixes(prefixes.browser)
-      transport.setLogLevelStyles(colors.browser)
+      output = globalScope.console
+      transportSource = BrowserTransport
```

Comment:
PascalCase `Transport` or `DefaultTransport` (or `TransportSource` – but what is the "source" in this? It is a constructor, not a factory.)